code standards updates

TanyaStere42 opened the pull request:
over 10.4 deployments, several new code standards have been proposed. also some reformatting for clarity

Atticus29 commented:
@holmbergius I'm seeing a lot of redundant headers.
E.g.:
<img width="1552" alt="Screen Shot 2022-06-08 at 10 10 07 PM" src="https://user-images.githubusercontent.com/2775448/172769043-8ba5e96e-4ceb-461f-9861-118383e5cfc4.png">
<img width="1552" alt="Screen Shot 2022-06-08 at 10 10 02 PM" src="https://user-images.githubusercontent.com/2775448/172769055-31ad6aa3-3151-4265-9ba3-773d7b0af559.png">

Not sure whether desired?

Atticus29 reviewed (approved):
LGTM other than the headers being redundant to the titles, as mentioned in the comment on the PR. Also, do we want the TBDs to go public? Or will Tanya or someone else be fleshing these out before deployment?

TanyaStere42 committed:
code standards updates

over 10.4 deployments, several new code standards have been proposed. also some reformatting for clarity

naknomum reviewed (approved):
looks good
